### PR TITLE
Issue/14 mobilenet

### PR DESCRIPTION
**Pull request by devon12stone**
Added:

- mobilenet ipynb
- confusion matrix code
- per-class accuracy code

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -8,7 +8,6 @@ dependencies:
   - pip
   - pytorch
   - torchvision
-  - cudatoolkit=10.2
```
Comment (James-Leslie) [inline environment.yml:11]: Can we make a separate `environment_cpu.yml` file?

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (James-Leslie) [inline .vscode/launch.json:1]: Could you add `.vscode` to our `.gitignore` file?

File: 1_mobilenet.ipynb [hunk 3]
```diff
@@ -0,0 +1,435 @@
+{
```
Comment (James-Leslie) [inline 1_mobilenet.ipynb:1]: Could you run the cells in this notebook so we can view the output in GitHub?

File: environment.yml (hunk 1 above)
Comment (devon12stone) [reply environment.yml:11]: done

File: .vscode/launch.json (hunk 2 above)
Comment (devon12stone) [reply .vscode/launch.json:1]: Done. How do I check if this works?

Comment (James-Leslie) [reply .vscode/launch.json:1]: when you do `$ git status` you won't see `.vscode` in your list of changed files. Also, in the VS Code editor, this file will appear greyed out.

Comment (James-Leslie) [reply .vscode/launch.json:1]: It looks like the `.vscode` folder is still being added in this pull request. So you will need to unstage them from the commit. I am not entirely sure how to do that though.
It's also super chilled to just delete them from master after this PR is merged.

**Pull request merged**